tox.ini

```diff
@@ -243,6 +243,12 @@ deps = {[black]deps}
 commands =
     black --diff wandb/ tests/ tools/
 
+[testenv:pyupgrade]
+basepython=python3
+deps=pyupgrade
+commands=
+    pyupgrade --py36-plus **/*.py
```

Comment:
at some point we will probably moving to py37... (maybe make it more generic?)

Reply:
i couldn't quickly figure out how to provide a default if nothing is passed to the tox -e command, so just hardcoded it here...